README.md

```diff
@@ -85,5 +85,16 @@ File for the requirements needed for the build process and the build script.
 ### os_dependencies
 When there is a need for additional OS dependencies to successfully build the wheel(s) on a specific platform and architecture, the `.sh` script in the `os_dependencies` directory can be adjusted.
 
+## Activity diagram
+The main file is `build-wheels-platforms.yml` which is scheduled to run twice a week to build Python wheels for any requirement of all [ESP-IDF]-supported versions.
```

Comment:
"scheduled to run twice a week" - I wonder if we would remember to correct this if we change that. It would be better to not include such details here. You could say that it is run periodically.